### PR TITLE
fix: fix the alignment issue between input box and description/unit value for the step input component.

### PR DESCRIPTION
**Pull request by manu-kr**
## Related Issue
Part of SAP/fundamental-ngx#5025

## Description
Changes contains to move move hard coded style from ngx to  style to fix the alignment issue between input box and description.

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [NA] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Comment by droshev** [comment]
@manu-kr can you update the title of the PR to be more meaningful with what the change is about?

**Comment by manu-kr** [comment]
> @manu-kr can you update the title of the PR to be more meaningful with what the change is about?

>@droshev i have update the PR, please let me know if still it require. changes.

File: src/step-input.scss [hunk 1]
```diff
@@ -84,6 +84,15 @@ $block: #{$fd-namespace}-step-input;
     }
   }
 
+  &__container {
+    display: flex;
+    align-items: center;
```
Comment (InnaAtanasova) [inline src/step-input.scss:89]: Hi @manu-kr , there are checkboxes in the PR template that are put there with a purpose, not just to waste people's time to check them when they open a PR. One of these checks is about mixins:

Screen Shot 2021-04-29 at 11 10 33 AM

In `_mixins.scss` file there's a mixin called `fd-flex-vertical-center` which is basically this:
Screen Shot 2021-04-29 at 11 13 33 AM

You can use it in your code.

File: src/step-input.scss [hunk 2]
```diff
@@ -84,6 +84,15 @@ $block: #{$fd-namespace}-step-input;
...
+
+    .fd-form-label {
```
Comment (InnaAtanasova) [inline src/step-input.scss:91]: You can't use this directly here as it's violating the self-contained rules.

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: src/step-input.scss (hunk 1 above)
Comment (manu-kr) [reply src/step-input.scss:89]: Hi @InnaAtanasova , Sorry i have missed this. I have reused this now.

File: src/step-input.scss (hunk 2 above)
Comment (manu-kr) [reply src/step-input.scss]: removed.

**Review by InnaAtanasova — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**